<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>login</class>
    <widget class="QWidget" name="login">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>280</width>
                <height>350</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>登录</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
                <spacer name="verticalSpacer_2">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>40</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="7,3">
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout_3">
                            <item>
                                <widget class="QLabel" name="label">
                                    <property name="text">
                                        <string>主机</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="ip_edit"/>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout_2">
                            <item>
                                <widget class="QLabel" name="label_2">
                                    <property name="text">
                                        <string>端口</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="prot_edit"/>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QLabel" name="label_3">
                            <property name="text">
                                <string>用户名</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>188</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QLineEdit" name="username_edit"/>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                        <widget class="QLabel" name="label_4">
                            <property name="text">
                                <string>密码</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer_2">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>188</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QLineEdit" name="password_edit"/>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                        <widget class="QCheckBox" name="remember_box">
                            <property name="text">
                                <string>记住密码</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer_3">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>188</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                        <spacer name="horizontalSpacer_6">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>98</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout">
                            <item>
                                <widget class="QPushButton" name="login_but">
                                    <property name="text">
                                        <string>登录</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer_5">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>98</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                </layout>
            </item>
            <item>
                <spacer name="verticalSpacer">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>30</height>
                        </size>
                    </property>
                </spacer>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
